sample-events/event-mapping.json

```diff
@@ -0,0 +1,365 @@
+      "name": "DynamoDbUpdate",
+      "command": "update",
+      "description": "DynamoDB Update",
+      "flags": [
+        {
+          "name": "region, r",
+          "usage": "The region the event should come from",
+          "value": "us-east-1"
+        }
+      ]
+    },
+    {
+      "name": "Kinesis",
+      "command": "kinesis",
+      "description": "Kinesis",
+      "flags": [
+        {
+          "name": "region, r",
+          "usage": "The region the event should come from",
+          "value": "us-east-1"
+        },
+        {
+          "name": "sequence, s",
+          "usage": "The Kinesis sequence number",
+          "value": "49545115243490985018280067714973144582180062593244200961"
+        },
+        {
+          "name": "data, d",
+          "usage": "he Kinesis message payload. There is no need to base64 this - sam will do this for you",
+          "value": "Hello, this is a test 123."
+        }
+      ]
+    },
+    {
+      "name": "KinesisFirehose",
+      "command": "firehose",
+      "description": "Kinesis Firehose",
+      "flags": []
+    },
+    {
+      "name": "KinesisFirehoseApacheLog",
+      "command": "firehose-apache-log",
+      "description": "Kinesis Firehose Apache Log",
+      "flags": []
+    },
+    {
+      "name": "KinesisFirehoseStreamAsSource",
+      "command": "firehose-stream-as-source",
+      "description": "Kinesis Firehose Stream As Source",
+      "flags": []
+    },
+    {
+      "name": "KinesisFirehoseSyslog",
+      "command": "firehose-syslog",
+      "description": "Kinesis Firehose Syslog",
+      "flags": []
+    },
+    {
+      "name": "RekognitionS3Request",
+      "command": "s3-request",
+      "description": "Rekognition S3 Request",
+      "flags": []
+    },
+    {
+      "name": "S3Delete",
+      "command": "delete",
+      "description": "S3 Delete",
+      "flags": [
+        {
+          "name": "region, r",
+          "usage": "The region the event should come from",
+          "value": "us-east-1"
+        },
+        {
+          "name": "bucket, b",
+          "usage": "The S3 bucket the event should reference",
+          "value": "example-bucket"
+        },
+        {
+          "name": "key, k",
+          "usage": "The S3 key the event should reference",
+          "value": "test/key"
+        }
+      ]
+    },
+    {
+      "name": "S3Put",
+      "command": "put",
+      "description": "S3 Put",
+      "flags": [
+        {
+          "name": "region, r",
+          "usage": "The region the event should come from",
+          "value": "us-east-1"
+        },
+        {
+          "name": "bucket, b",
+          "usage": "The S3 bucket the event should reference",
+          "value": "example-bucket"
+        },
+        {
+          "name": "key, k",
+          "usage": "The S3 key the event should reference",
+          "value": "test/key"
+        }
+      ]
+    },
+    {
+      "name": "ScheduledEvent",
+      "command": "scheduled-event",
```

Comment:
This was called `schedule`. Is it possible to change the name of all existing events to match the original command for back compat?